docs/plot.py

```diff
@@ -0,0 +1,17 @@
+import matplotlib.pyplot as plt
+from astropy.coordinates import SkyCoord
+
+from hips.draw.simple import make_sky_image
+from hips.tiles import HipsSurveyProperties
+from hips.utils import WCSGeometry
+
+url = 'http://alasky.unistra.fr/DSS/DSS2Merged/properties'
+hips_survey = HipsSurveyProperties.fetch(url)
+geometry = WCSGeometry.create_simple(
+    skydir=SkyCoord(0, 0, unit='deg', frame='galactic'),
+    width=2000, height=1000, fov="3 deg",
+    coordsys='galactic', projection='AIT'
+)
+data = make_sky_image(geometry=geometry, hips_survey=hips_survey, tile_format='jpg')
```

Comment:
I would suggest to just add the "fits" plotting example that is working here for now.
JPEG drawing isn't working yet, and is probably best for a separate PR, no?